django_short_urls/views.py

```diff
@@ -162,6 +162,10 @@ def new(request):  # pylint: disable=too-many-branches
                 status=HTTP_BAD_REQUEST,
                 message="%s may not contain a '/' character." % key)
 
+    statsd.increment(
+        'workforus.new',
+        tags=['prefix:' + unicode(params['prefix']), 'is_secure:' + unicode(request.is_secure())])
```

Comment:
But the SSL information is irrelevant to "new" specifically − I feel it should be backward: why don't we monitor all requests to the shortener for their SSL status, tagging whether they are "get" or "API new" calls?